packages/desktop-client/src/components/reports/reports/CalendarCard.tsx

```diff
@@ -0,0 +1,543 @@
+    useState<string>('MMMM yyyy');
+
+  useEffect(() => {
+    if (data) {
+      setMonthNameFormats(
+        Array(data.calendarData.length).map(() => 'MMMM yyyy'),
+      );
+    } else {
+      setMonthNameFormats([]);
+    }
+  }, [data]);
+
+  useEffect(() => {
+    if (monthNameFormats.length) {
+      setSelectedMonthNameFormat(
+        monthNameFormats.reduce(
+          (a, b) => ((a?.length ?? 0) <= (b?.length ?? 0) ? a : b),
+          'MMMM yyyy',
+        ),
+      );
+    } else {
+      setSelectedMonthNameFormat('MMMM yyyy');
+    }
+  }, [monthNameFormats]);
+
+  const calendarLenSize = useMemo(() => {
+    if (!data) {
+      return 0;
+    }
+
+    return data?.calendarData.length;
+  }, [data]);
+
+  return (
+    <ReportCard
+      isEditing={isEditing}
+      to={`/reports/calendar/${widgetId}`}
+      menuItems={[
+        {
+          name: 'rename',
+          text: t('Rename'),
+        },
+        {
+          name: 'remove',
+          text: t('Remove'),
+        },
+      ]}
+      onMenuSelect={item => {
+        switch (item) {
+          case 'rename':
+            setNameMenuOpen(true);
+            break;
+          case 'remove':
+            onRemove();
+            break;
+          default:
+            throw new Error(`Unrecognized selection: ${item}`);
+        }
+      }}
+    >
+      <View
+        ref={el => el && cardRef(el)}
+        style={{ flex: 1, margin: 2, overflow: 'hidden', width: '100%' }}
+      >
+        <View style={{ flexDirection: 'row', padding: 20, paddingBottom: 0 }}>
+          <View style={{ flex: 1, marginBottom: -5 }}>
+            <ReportCardName
+              name={meta?.name || t('Calendar')}
+              isEditing={nameMenuOpen}
+              onChange={newName => {
+                onMetaChange({
+                  ...meta,
+                  name: newName,
+                });
+                setNameMenuOpen(false);
+              }}
+              onClose={() => setNameMenuOpen(false)}
+            />
+          </View>
+          <View style={{ textAlign: 'right' }}>
+            <Block
+              style={{
+                ...styles.mediumText,
+                fontWeight: 500,
+              }}
+            >
+              <Tooltip
+                content={
+                  <View style={{ lineHeight: 1.5 }}>
+                    <View
+                      style={{
+                        display: 'grid',
+                        gridTemplateColumns: '70px 1fr',
+                        gridAutoRows: '1fr',
+                      }}
+                    >
+                      {totalIncome !== 0 && (
+                        <>
+                          <View
+                            style={{
+                              textAlign: 'right',
+                              marginRight: 4,
+                            }}
+                          >
+                            Income:
```

Comment:
```suggestion
                            <Trans>Income:</Trans>
```

Reply:
@jfdoming how would we do the privacy filter on this example?
`<Trans>Income: {{ incomeValue }}</Trans>`

Reply:
I think `<Trans>Income: <Privacy filter>{{ incomeValue }}</PrivacyFilter></Trans>` should work

Reply:
And generate a string like "Income: <1>{incomeValue}</1>"

Reply:
@jfdoming  I kept @matt-fidd suggestion since Income and the value are not even on the same level:
![image](https://github.com/user-attachments/assets/a5f406d9-22bb-4094-b84b-076fd6457f62)

It may open for confusing context translations, but It keeps the layout as it is

Reply:
I see. Maybe in the future we may need to update, but for now I guess this is fine